Feature branch

asadmisrani opened the pull request:
Hello Microverse.
This is a small project about portfolio mobile first.
I made this with HTML CSS and Flexbox.
I have did all the requirements such as linter error, gitignore file, test md file, README file with template given in microverse.

asadmisrani committed:
all files added

asadmisrani committed:
readme file modified

asadmisrani committed:
css file modified

asadmisrani committed:
css file modified

asadmisrani committed:
css file modified

asadmisrani committed:
css file modified

asadmisrani committed:
css file modified

DuaneDave commented on index.html:
```diff
@@ -0,0 +1,205 @@
+<body>
+    <section class="container">
+    <header>
+        <a href="">My Logo</a>
```
[OPTIONAL]  Consider adding an # for the `href` attribute on the logo so it leads nowhere or better still you can point it to the headline in order to avoid reloading the page whenever it is clicked.

Like this
```````
 <a href="#">My Logo</a>
```````


```````
 <a href="#some id that point to the headline">My Logo</a>
```````

asadmisrani replied:
Hello DuaneDave.
I did all the things that you highlighted in a comment.
I commit first latter capital and git commit.
I made nav items clickable with a # tag.
I also make Figma color and # tag in let's connect.
Thank you

DuaneDave commented on index.html:
```diff
@@ -0,0 +1,205 @@
+            <li>Contact</li>
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
+    </header>
```
[OPTIONAL] The nav items are clickable links that enable a user to navigate specific areas of the website. Kindy render the items as an `a` tag in order to make navigating the portfolio easier.

Like this

````````
<li><a>your items here</a></li>
````````


[REQUIRED] Furthermore, I recommend that the spaces at the top of the navbar are reduced and the hamburger should be styled to match the Figma design in order to make the front end more organized and more visually appealing.


YOUR VERSION

![bad nav](https://user-images.githubusercontent.com/84930355/212210010-c1c65cbd-e240-44aa-ac46-adbd2b4312bb.PNG)


FIGMA DESIGN

![good nav](https://user-images.githubusercontent.com/84930355/212210135-580a9a67-7d82-48e3-bcca-188e21db2577.PNG)

asadmisrani replied:
Hello DuaneDave.
I made nav items clickable with a # tag.

DuaneDave commented on index.html:
```diff
@@ -0,0 +1,205 @@
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
+    </header>
+    <div class="headline">
+        <h1>I’m Asad Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title">LET’S CONNECT</p>
+        <ul>
+            <li class="social" ><a href=""><img src="assets/twitter.svg" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Linkedin icon.svg" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
+        </ul>
+    </div>
+    </section>
```
The following are the issues found in the headline section:

- Kindly use the correct color for the lets connect in order to make it match the Figma design and make it more visually appealing.

- I also suggest that the let's connect should be a clickable link because when it is clicked it should lead to the contact form which will make the page easily navigatable.

- To make the UI look nicer, consider adding the background images on the headline section and also the rounded edge at the bottom left of the section.


Notice the highlighted section  in the screenshot below


YOUR DESIGN

![no nothing](https://user-images.githubusercontent.com/84930355/212212782-628bf8db-7c33-4aa8-8ef9-fd9e9f7d1b8a.PNG)


FIGMA DESIGN

![good bg](https://user-images.githubusercontent.com/84930355/212213331-6ff9fc14-f0e3-484d-81a1-567b6d55873c.PNG)

DuaneDave reviewed (changes requested):
Hi @asadmisrani 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good feature branch name 👍
✔️ Meaningful commit history 👍 
✔️ Well structured code 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please make sure to start a commit message with a capital letter in order to follow best practices for a professional commit.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani committed:
Css file modified

asadmisrani committed:
Css file modified

asadmisrani committed:
Css file modified

asadmisrani committed:
Css file modified

MaganaAsiati commented on index.html:
```diff
@@ -0,0 +1,205 @@
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        <nav class="flex-nav"><button>☰</button></nav>
```
- Kindly make sure that you design your hamburger menu as it shows in the Figma design. Please fix that as required by the previous reviewer :+1:

asadmisrani replied:
I solved the issue and make the hamburger menu as shown in the Figma design.

AbayomiOlaoye replied:
- Good job, @asadmisrani 👍
- [ ] Do you agree that a `dead link` is no link? I think you actually add your `index.html` as reference in the `href` attribute. This is improves access to your site and offers good user experience because visitors can always find their way back to your homepage in case they get lost somewhere and needed to confirm some info again.
- [ ] To make you hamburger button look like it is on the template, please consider removing the `button` tag so that the menu button is just in your `nav` tag.

MaganaAsiati commented on index.html:
```diff
@@ -0,0 +1,205 @@
+            </li>
+            <li id="lang3"><img src="./assets/css.png" alt="">
+            <p>css</p>
+            </li>
+            <li id="framework">
+                <h2>Frameworks</h2>
+                <p>></p>
+            </li>
+            <li id="skills">
+                <h2>Skills</h2>
+                <p>></p>
+            </li>
+        </ul>
+    </section>
+    <section class="contact">
+        <footer class="contact-form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xlevdjlz" method="POST">
+            <ul>
+                <li>
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" placeholder="Your Name" maxlength="30" required>
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="Enter your email" required>
+                </li>
+                <li>
+                    <label for="message"></label>
+                    <textarea id="message" placeholder="Enter your message here" maxlength="500" required></textarea>
+                </li>
+                <li><button type="submit"><span> Get in touch </span></button></li>
+                <li>
+                    <div class="shape"><img src="./assets/Shape.png" alt=""></div>
+                </li>
+            </ul>
+        </form>
+        </footer>
+    </section>
```
- [ ] [ Optional ] Kindly remove  the card section and the contact section  as they  have not yet part of the requirements
:+1:

AbayomiOlaoye replied:
- [ ] [OPTIONAL] Please consider the suggestion of my colleague. Removing this will allow you to work faster and more conveniently.

MaganaAsiati commented on index.html:
```diff
@@ -0,0 +1,205 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
```
- Kindly add The background image  to your home page as it is in the  design of the template you chose  it will make your design stick more to the Figma design :+1: 
- Kindly also the lets us connect button must not be underlined please fix that  sot that it will make your design stick to the Figma design :+1: 

## Your design

![image](https://user-images.githubusercontent.com/95297251/212363879-001cf0e4-63bf-4599-ab2f-ac7c7b488fce.png)

## Figma Design 

![image](https://user-images.githubusercontent.com/95297251/212364356-7b8fb209-c5d1-4f34-8fa6-cc8d77ae4b67.png)

asadmisrani replied:
I have made changes in the background image of my home page as you mention me

AbayomiOlaoye replied:
- [ ] I must commend your honest effort but it seems nothing has changed.

- Here is a snapshot of what your page looks like now: 

![image](https://user-images.githubusercontent.com/99715640/212410510-2daf6669-9d25-4785-a076-3d6cc30bcfb8.png)

- [ ] I think a major reason why your `background-image` is not showing is because you are trying to import an image file that you don't currently have in your `assets` folder.

- Check this 👇 
![image](https://user-images.githubusercontent.com/99715640/212411444-d82eb545-006f-452c-876b-c9cabae91d3f.png)
- [ ] Kindly check your file name
- [ ] Also, as the previous reviewer indicated, please check your `font` properties and `text-decoration` property on `LET'S CONNECT` element.

MaganaAsiati commented on index.html:
```diff
@@ -0,0 +1,205 @@
+    <div class="headline">
+        <h1>I’m Asad Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title"><a href="#">LET’S CONNECT</a></p>
```
- [ ]  [OPTIONAL ] Kindly   use the` <p> `tag only for paragraph  LET'S CONNECT is not a paragraph please do not use `<p> ` tag for that :+1:

asadmisrani replied:
I have change p tag in LET'S CONNECT as well

MaganaAsiati reviewed (changes requested):
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @asadmisrani , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-gif-4](https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif)

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

asadmisrani committed:
Css file modified

asadmisrani reviewed (commented):
I have made all the changes that you highlighted to me.

AbayomiOlaoye commented on style.css:
```diff
@@ -0,0 +1,704 @@
+  header ul { display: none; }
+
+  header a {
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```
- [ ] [OPTIONAL] Kindly remove this line that disables linter code check. Linter is a github action that ensures that codes are standard. Getting familiar with these practices helps you to avoid bad syntax and substandard practices.

AbayomiOlaoye commented on style.css:
```diff
@@ -0,0 +1,704 @@
+
+  .headline h1 {
+    font-style: normal;
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```
- [ ] [OPTIONAL] Kindly remove this line that disables linter code check. Linter is a github action that ensures that codes are standard. Getting familiar with these practices helps you to avoid bad syntax and substandard practices.

AbayomiOlaoye commented on style.css:
```diff
@@ -0,0 +1,704 @@
+  .flex-nav { color: #6070ff; }
+
+  .headline {
+    background-image: url(./assets/headershapesmobile.png);
```
- Kindly check to be sure that you have this in the folder. If you don't, like I think you don't, try downloading it or rename the file.

AbayomiOlaoye commented on style.css:
```diff
@@ -0,0 +1,704 @@
+
+  .headline p {
+    font-style: normal;
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```
- [ ] [OPTIONAL] Kindly remove this line that disables linter code check. Linter is a github action that ensures that codes are standard. Getting familiar with these practices helps you to avoid bad syntax and substandard practices.

AbayomiOlaoye reviewed (changes requested):
Hi @asadmisrani 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so_close](https://user-images.githubusercontent.com/99715640/211872359-9a4d36f7-0386-4000-82a4-c274b06a5ce4.gif)


 ## Highlights
- Good PR message ✅ 
- Great commit messages ✅ 
- No linter error ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- You can also consider:
- [ ] Adding a descriptive README file to your project according to [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani committed:
Style file is modified

asadmisrani commented:
Thank you for laying me know.1- I add my index.html as a reference in the href attribute for a good user experience.2- I have removed the button tag from the menu button.3- I have added my missing jpg file in the assets folder on my desktop.4- I removed all files of Linter is a GitHub action from the CSS file.Now kindly recheck and approve my changes fast because I don't have enough time I have more work left in Microverse.Thank you, my friend.

asadmisrani commented:
Hi  *AbayomiOlaoye <https://github.com/AbayomiOlaoye>*
Thank you for laying me know.
1- I add my index.html as a reference in the href attribute for a good user
experience.
2- I have removed the button tag from the menu button.
3- I have added my missing jpg file in the assets folder on my desktop.
4- I removed all files of Linter is a GitHub action from the CSS file.
Now kindly recheck and approve my changes fast because I don't have enough
time I have more work left in Microverse.
Thank you, my friend.


On Sat, Jan 14, 2023 at 1:31 AM Olaoye Oluwatoyin Abayomi <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @asadmisrani <https://github.com/asadmisrani> 👋 ,
>
> Great work on making the changes requested by a previous reviewer 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
>
> [image: so_close]
> <https://user-images.githubusercontent.com/99715640/211872359-9a4d36f7-0386-4000-82a4-c274b06a5ce4.gif>
> Highlights
>
>    - Good PR message ✅
>    - Great commit messages ✅
>    - No linter error ✅
>
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you to take them into account as they can
> make your code better. Some of them were simply missed by the previous
> reviewer and addressing them will really improve your application.*
>
>    - You can also consider:
>    - Adding a descriptive README file to your project according to this
>    template <https://github.com/microverseinc/readme-template> but
>    remember to customize it to your project.
>    Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In style.css
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069990461>
> :
>
> > ***@***.*** (min-width: 320px) {
>
> +  body {
>
> +    margin: 0;
>
> +    overflow-x: hidden;
>
> +  }
>
> +  .container { margin: 52px 24px 0 24px; }
>
> +
>
> +  header {
>
> +    display: flex;
>
> +    justify-content: space-between;
>
> +    align-items: baseline;
>
> +  }
>
> +  header ul { display: none; }
>
> +
>
> +  header a {
>
> +    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
>
>
>
>    - [OPTIONAL] Kindly remove this line that disables linter code check.
>    Linter is a github action that ensures that codes are standard. Getting
>    familiar with these practices helps you to avoid bad syntax and substandard
>    practices.
>
> ------------------------------
>
> In style.css
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069990685>
> :
>
> > +  }
>
> +
>
> +  .flex-nav { color: #6070ff; }
>
> +
>
> +  .headline {
>
> +    background-image: url(./assets/headershapesmobile.png);
>
> +    background-color: #fff;
>
> +    display: flex;
>
> +    flex-direction: column;
>
> +    background-size: cover;
>
> +    padding-top: 80.8px;
>
> +  }
>
> +
>
> +  .headline h1 {
>
> +    font-style: normal;
>
> +    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
>
>
>
>    - [OPTIONAL] Kindly remove this line that disables linter code check.
>    Linter is a github action that ensures that codes are standard. Getting
>    familiar with these practices helps you to avoid bad syntax and substandard
>    practices.
>
> ------------------------------
>
> In style.css
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069992533>
> :
>
> > +
>
> +  header a {
>
> +    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
>
> +    font-family: 'Poppins';
>
> +    text-decoration: none;
>
> +    font-style: normal;
>
> +    font-weight: 700;
>
> +    font-size: 18px;
>
> +    line-height: 20px;
>
> +    color: #6070ff;
>
> +  }
>
> +
>
> +  .flex-nav { color: #6070ff; }
>
> +
>
> +  .headline {
>
> +    background-image: url(./assets/headershapesmobile.png);
>
>
>
>    - Kindly check to be sure that you have this in the folder. If you
>    don't, like I think you don't, try downloading it or rename the file.
>
> ------------------------------
>
> In style.css
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069992781>
> :
>
> > +  }
>
> +
>
> +  .headline h1 {
>
> +    font-style: normal;
>
> +    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
>
> +    font-family: 'Poppins';
>
> +    font-weight: 700;
>
> +    font-size: 40px;
>
> +    line-height: 52px;
>
> +    color: #172b4d;
>
> +    margin-bottom: 12px;
>
> +  }
>
> +
>
> +  .headline p {
>
> +    font-style: normal;
>
> +    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
>
>
>
>    - [OPTIONAL] Kindly remove this line that disables linter code check.
>    Linter is a github action that ensures that codes are standard. Getting
>    familiar with these practices helps you to avoid bad syntax and substandard
>    practices.
>
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069972363>
> :
>
> > +    <meta charset="UTF-8">
>
> +    <meta http-equiv="X-UA-Compatible" content="IE=edge">
>
> +    <meta name="viewport" content="width=device-width, initial-scale=1.0">
>
> +    <title>Asad Shareef Protfolio</title>
>
> +    <link rel="stylesheet" href="style.css">
>
> +</head>
>
> +<body>
>
> +    <section class="container">
>
> +    <header>
>
> +        <a href="#">My Logo</a>
>
> +        <ul>
>
> +            <li><a href="#">Protfolio</a> </li>
>
> +            <li><a href="#">About</a></li>
>
> +            <li><a href="#">Contact</a></li>
>
> +        </ul>
>
> +        <nav class="flex-nav"><button>☰</button></nav>
>
>
>
>    - Good job, @asadmisrani <https://github.com/asadmisrani> 👍
>    - Do you agree that a dead link is no link? I think you actually add
>    your index.html as reference in the href attribute. This is improves
>    access to your site and offers good user experience because visitors can
>    always find their way back to your homepage in case they get lost somewhere
>    and needed to confirm some info again.
>    - To make you hamburger button look like it is on the template, please
>    consider removing the button tag so that the menu button is just in
>    your nav tag.
>
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069974296>
> :
>
> > +                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
>
> +                <ul class="languages">
>
> +                    <li>html</li>
>
> +                    <li>css</li>
>
> +                    <li>javaScript</li>
>
> +                </ul>
>
> +            <button>See Project</button>
>
> +            </div>
>
> +            </div>
>
> +        <div class="cards card4">
>
> +            <img src="./assets/Snapshoot3.png" alt="">
>
> +            <div class="card-2">
>
> +            <div class="card1">
>
> +                <h2>Multi-Post Stories</h2>
>
> +                <ul>
>
> +                    <li class="canboy-m">CANOPY</li>
>
> +                    <li class="facebook-d">FACEBOOK</li>
>
> +                    <li>⬤</li>
>
> +                    <li class="backend-m">BackEnd Dev</li>
>
> +                    <li class="full-d">Full Stack Dev</li>
>
> +                    <li>⬤</li>
>
> +                    <li>2015</li>
>
> +                </ul>
>
> +                </div>
>
> +                <p class="card4p-m">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
>
> +                <p class="card-d"> Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends. </p>
>
> +                <ul class="languages">
>
> +                    <li>html</li>
>
> +                    <li class="ruby-d">Ruby on rails</li>
>
> +                    <li>css</li>
>
> +                    <li>javaScript</li>
>
> +                </ul>
>
> +                <button>See Project</button>
>
> +            </div>
>
> +            </div>
>
> +    </section>
>
> +    <section class="about">
>
> +        <div class="about-social">
>
> +        <h2>About <br> Myself</h2>
>
> +        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
>
> +        <p id="social-title-1">LET’S CONNECT</p>
>
> +        <ul>
>
> +            <li class="social" ><a href=""><img src="assets/twitter.png" alt=""></a></li>
>
> +            <li class="social" ><a href=""><img src="assets/Linkedinicon.png" alt=""></a></li>
>
> +            <li class="social" ><a href=""><img src="assets/Vector.png" alt=""></a></li>
>
> +            <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
>
> +            <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
>
> +        </ul>
>
> +        <button>Get my resume</button>
>
> +        </div>
>
> +        <ul class="about-list">
>
> +            <li id="lang">
>
> +                <h2>Languages</h2>
>
> +                <p>˅</p>
>
> +            </li>
>
> +            <li id="lang1"><img src="./assets/js.png" alt="">
>
> +            <p>javaScript</p>
>
> +            </li>
>
> +            <li id="lang2"><img src="./assets/html.png" alt="">
>
> +            <p>html</p>
>
> +            </li>
>
> +            <li id="lang3"><img src="./assets/css.png" alt="">
>
> +            <p>css</p>
>
> +            </li>
>
> +            <li id="framework">
>
> +                <h2>Frameworks</h2>
>
> +                <p>></p>
>
> +            </li>
>
> +            <li id="skills">
>
> +                <h2>Skills</h2>
>
> +                <p>></p>
>
> +            </li>
>
> +        </ul>
>
> +    </section>
>
> +    <section class="contact">
>
> +        <footer class="contact-form">
>
> +        <h2>Contact me</h2>
>
> +        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
>
> +        <form action="https://formspree.io/f/xlevdjlz" method="POST">
>
> +            <ul>
>
> +                <li>
>
> +                    <label for="name"></label>
>
> +                    <input type="text" name="user_name" id="name" placeholder="Your Name" maxlength="30" required>
>
> +                </li>
>
> +                <li>
>
> +                    <label for="email"></label>
>
> +                    <input type="email" name="user_email" id="email" placeholder="Enter your email" required>
>
> +                </li>
>
> +                <li>
>
> +                    <label for="message"></label>
>
> +                    <textarea id="message" placeholder="Enter your message here" maxlength="500" required></textarea>
>
> +                </li>
>
> +                <li><button type="submit"><span> Get in touch </span></button></li>
>
> +                <li>
>
> +                    <div class="shape"><img src="./assets/Shape.png" alt=""></div>
>
> +                </li>
>
> +            </ul>
>
> +        </form>
>
> +        </footer>
>
> +    </section>
>
>
>
>    - [OPTIONAL] Please consider the suggestion of my colleague. Removing
>    this will allow you to work faster and more conveniently.
>
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/desktop.github.io/pull/1#discussion_r1069987388>
> :
>
> > @@ -0,0 +1,205 @@
>
> +<!DOCTYPE html>
>
> +<html lang="en">
>
> +<head>
>
> +    <meta charset="UTF-8">
>
> +    <meta http-equiv="X-UA-Compatible" content="IE=edge">
>
> +    <meta name="viewport" content="width=device-width, initial-scale=1.0">
>
> +    <title>Asad Shareef Protfolio</title>
>
> +    <link rel="stylesheet" href="style.css">
>
> +</head>
>
> +<body>
>
> +    <section class="container">
>
>
>
>    -
>
>    I must commend your honest effort but it seems nothing has changed.
>    -
>
>    Here is a snapshot of what your page looks like now:
>
> [image: image]
> <https://user-images.githubusercontent.com/99715640/212410510-2daf6669-9d25-4785-a076-3d6cc30bcfb8.png>
>
>    -
>
>    I think a major reason why your background-image is not showing is
>    because you are trying to import an image file that you don't currently
>    have in your assets folder.
>    -
>
>    Check this 👇
>    [image: image]
>    <https://user-images.githubusercontent.com/99715640/212411444-d82eb545-006f-452c-876b-c9cabae91d3f.png>
>    -
>
>    Kindly check your file name
>    -
>
>    Also, as the previous reviewer indicated, please check your font
>    properties and text-decoration property on LET'S CONNECT element.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/desktop.github.io/pull/1#pullrequestreview-1248403863>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKBUBVN6YC5A234KEO3WSG3SNANCNFSM6AAAAAATZWI4BA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,206 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="container">
```
- [ ] Good job adding the background image as suggested by the previous reviewer 👍🏽 .but then, the background doesn't look exactly like how it is on the figma design. Kindly ensure that your design matches the figma design 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212433540-84297bec-8d77-4399-b9f3-cea797a5b307.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212433964-71fc72f4-367a-4edd-b9b1-140d850b6786.png)

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,206 @@
+        <h1>I’m Asad Glad
+            to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 id="social-title"> <a href="#"> LET’S CONNECT </a></h3>
```
- [ ] Also, as the previous reviewer indicated, please check your font properties, color, and text-decoration property on `LET'S CONNECT` element. Ensure it matches the figma design

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212434156-67b8954d-5589-47d1-bc6d-17ba9b825bf9.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212434205-73f78299-50e7-473f-8b65-532174cd7fa5.png)

victorokoroji commented on style.css:
```diff
@@ -0,0 +1,687 @@
+  }
+
+  .headline {
+    /* display: flex;
+    background-image: url(./assets/desktop/desktopHeader.png);
+    padding: 288px 260px 255px 260px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    gap: 12px; */
```
[OPTIONAL]

- KIndly ensure you remove all commented codes from your codebase so that your code can be clean and readable

victorokoroji reviewed (changes requested):
Hi @asadmisrani, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- I will strongly recommend that you always write a descriptive branch name and pull request title describing changes that you introduced or the feature you are working on. Your branch name and pull request title don't represent what you are currently working on, making your repo unprofessional. Kindly improve on this because its a reason for the invalidation of your project.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
